### DISCUSSION TITLE
Error when importing ---- import { createWeb3Modal, defaultWagmiConfig } from '@web3modal/wagmi/react'

### DISCUSSION THREAD

**Discussion by LokeshATR** [WalletConnect/walletconnect-monorepo#3684] [Developer Support] [answered]
After following the DOCS on https://docs.walletconnect.com/2.0/web3modal/v3/react/about for creating basic react project, we are facing the error mentioned below.
![Screenshot from 2023-09-20 18-23-50](https://github.com/WalletConnect/walletconnect-monorepo/assets/142476456/a9381ac0-71e5-4873-ab39-33f263e4704a)

ERROR in ./node_modules/@web3modal/scaffold/dist/esm/src/client.js 106:52-79
Module not found: Error: Can't resolve './modal/w3m-modal' in '/data/react/shido_frontend/node_modules/@web3modal/scaffold/dist/esm/src'
Did you mean 'index.js'?
BREAKING CHANGE: The request './modal/w3m-modal' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

Thanks in advance,
lokeshbhavsar127@gmail.com

**Comment by glitch-txs** [comment, marked as answer]
could you try updaing to the lastest beta version?
